CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -2,6 +2,21 @@
 
 *Some deprecated APIs have been removed in Antrea v2.0. Before upgrading, please read these [guidelines](https://github.com/antrea-io/antrea/blob/release-2.0/docs/versioning.md#upgrading-from-antrea-v1-to-antrea-v2) carefully.*
 
+## 2.0.1 - 2024-06-14
+
+### Changed
+
+- Update CNI binaries version to v1.4.1. ([#6334](https://github.com/antrea-io/antrea/pull/6334), [@antoninbas])
+
+### Fixed
+
+- Remove unexpected `altname` after rename interface to avoid failure when moving host interface to OVS bridge. ([#6321](https://github.com/antrea-io/antrea/pull/6321), [@gran-vmv])
```

Comment:
```suggestion
- Remove unexpected `altname` after renaming interface to avoid failure when moving host interface to OVS bridge. ([#6321](https://github.com/antrea-io/antrea/pull/6321), [@gran-vmv])
```

Reply:
Updated, I think we can go ahead without rerun the e2e tests.